upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -623,6 +648,16 @@ do_upgrade_csm_chart cray-spire sysmgmt.yaml
 do_upgrade_csm_chart cray-tapms-crd sysmgmt.yaml
 do_upgrade_csm_chart cray-tapms-operator sysmgmt.yaml
 
+# Running the rollout restart script to restart the required resources in istio-injection=enabled namespaces.
```

Comment:
Please put rollout restart into wrapping block to ensure proper logging and idempotency:

    state_name="RESTART_SERVICES_REFRESH_ISTIO"
    state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
    if [[ ${state_recorded} == "0" && $(hostname) == "${PRIMARY_NODE}" ]]; then
      echo "====> ${state_name} ..." | tee -a "${LOG_FILE}"
      {
      ...
      } >> "${LOG_FILE}" 2>&1
      record_state "${state_name}" "$(hostname)" | tee -a "${LOG_FILE}"
    else
      echo "====> ${state_name} has been completed" | tee -a "${LOG_FILE}"
    fi